acknowledge_license_cert.html

```diff
@@ -11,28 +11,35 @@
 
 <h3>License information</h3>  
 <div class="paragraph">   
-Unless otherwise stated, this code is subject to the terms of the <a href="license.txt">
-GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here
- for your convenience.
+To be announced
 </div>
 <h3>Websites</h3>
 <div class="paragraph">
-<a href="http://www.open-emr.org/">LibreEHR</a> - The LibreEHR website: <a href="http://www.open-emr.org/">www.open-emr.org</a><br>
-<a href="http://www.oemr.org/">OEMR</a> - The OEMR organization website: <a href="http://www.oemr.org/">www.oemr.org</a>
+<a href="http://www.libreehr.org/">LibreEHR</a> - The LibreEHR website: <a href="http://www.libreehr.org/">www.libreehr.org</a><br>
+<a href="http://www.libreehr.org/">LibreEHR</a> - The LibreEHR organization website: <a href="http://www.libreehr.org/">www.libreehr.org</a>
+</div>
+
+<h3>Special Thanks</h3>
+<div class="paragraph">
+To all the contributers at open-emr.org for all the years of work they put in to their product which LibreEHR is forked from.<br> 
+To see all the contributers you can visit their acknowledgements page at open-emr.org.<br>
+Thanks Guys and Gals!!!
+<ul>
+<li>Managed and provided by LibreEHR.org  - <a href="http://www.libreehr.org">www.libreehr.org</a></li>
+</ul>
 </div>
 
 <h3>Certification</h3>
 <div class="paragraph">
-This EHR Module is 2014 Edition compliant and has been certified by an ONC-ACB in accordance with the applicable certification criteria adopted by the Secretary of Health and Human Services. This certification does not represent an endorsement by the U.S. Department of Health and Human Services or guarantee the receipt of incentive payments.
+Up and coming
 <ul>
-<li>Managed and provided by OEMR.org 501c3 - <a href="http://www.oemr.org">www.oemr.org</a></li>
-<li>Certified by InfoGard - <a href="http://www.infogard.com">www.infogard.com</a></li>
+<li>Managed and provided by LibreEHR.org  - <a href="http://www.libreehr.org">www.libreehr.org</a></li>
```

Comment:
Just let me Know

      From: Tony McCormick <notifications@github.com>
 To: LibreEHR/LibreEHR <LibreEHR@noreply.github.com> 
Cc: Terry Hill <teryhill@yahoo.com>; Author <author@noreply.github.com>
 Sent: Thursday, August 11, 2016 8:21 PM
 Subject: Re: [LibreEHR/LibreEHR] Changes to the acknowledgments page (#80)
   
In acknowledge_license_cert.html:>  <ul>
> -<li>Managed and provided by OEMR.org 501c3 - <a href="http://www.oemr.org">www.oemr.org</a></li>
> -<li>Certified by InfoGard - <a href="http://www.infogard.com">www.infogard.com</a></li>
> +<li>Managed and provided by LibreEHR.org  - <a href="http://www.libreehr.org">www.libreehr.org</a></li>
We will need to add LibreHealth.io and the Software Freedom Conservancy bylines when I get the official text.—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.